sdk/appconfiguration/app-configuration/src/appConfigurationClient.ts

```diff
@@ -142,30 +142,70 @@ export class AppConfigurationClient {
     options: GetConfigurationSettingOptions = {}
   ): Promise<GetConfigurationSettingResponse> {
     return await this.spanner.trace("getConfigurationSetting", options, async (newOptions) => {
-      const response = (await this.client.getKeyValue(id.key, {
+      const originalResponse = await this.client.getKeyValue(id.key, {
         label: id.label,
         select: newOptions.fields,
         ...newOptions,
-        ...checkAndFormatIfAndIfNoneMatch(newOptions)
-      })) as GetConfigurationSettingResponse;
+        ...checkAndFormatIfAndIfNoneMatch(id, newOptions)
+      });
+
+      const response: GetConfigurationSettingResponse = {
+        ...originalResponse,
+        _response: originalResponse._response,
```

Comment:
we have to do this because `_response` isn't enumerable, right? Why are we doing that again?

Reply:
If you're asking why I'm copying the object - I added in a new top level field (`statusCode`) for this and deleteConfigurationSetting, both of which can return these throw-happy ConfigurationSetting objects. Since the user has to check the status code to get an idea of what's going on we moved it to the top to make that easier.

`statusCode` is non-nullable and thus isn't compatible with the underlying KeyValueResponse from the generated code. Thus this copy.

Now if you're asking me why _response isn't enumerable, that I don't know. It surprised me the other day as well. A question for @bterlson, probably.